components/nexusoperations/executors.go

```diff
@@ -76,41 +76,34 @@ func RegisterExecutor(
 	options ActiveExecutorOptions,
 ) error {
 	exec := activeExecutor{options}
-	if err := hsm.RegisterImmediateExecutors(
+	if err := hsm.RegisterImmediateExecutor[InvocationTask](
```

Comment:
This should be inferred.

Reply:
It it, but I just wanted to make it explicit.